changelog.txt

```diff
@@ -1,5 +1,9 @@
 *** WooCommerce Payments Changelog ***
 
+= 2.0.0 - 2021-xx-xx =
+
```

Comment:
Do we need an empty line here?

Reply:
Not really :) fixed in https://github.com/Automattic/woocommerce-payments/pull/1241/commits/232948a6736f73a228639a3e18f001edbecd3250